src/cryptography/hazmat/bindings/openssl/_conditional.py

```diff
@@ -282,6 +282,7 @@
         "X509_VERIFY_PARAM_set1_ip",
         "X509_VERIFY_PARAM_set1_ip_asc",
         "X509_VERIFY_PARAM_set_hostflags",
+        "X509_VERIFY_PARAM_free",
```

Comment:
You don't need this here since `X509_VERIFY_PARAM_free` isn't part of the new 1.0.2 pieces (it is present long before our minimum supported 1.0.0).